storey/flow.py

```diff
@@ -629,3 +664,150 @@ def build_flow(steps):
             cur_step.to(next_step)
             cur_step = next_step
     return steps[0]
+
+
+class V3ioTable(NeedsV3ioAccess):
+    def __init__(self, table, webapi=None, access_key=None):
```

Comment:
I prefer leaving the table out as a method argument to keep it flexible. Otherwise, working with multiple tables becomes inconvenient.